lib/CUDD/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
 set(SOURCES
        cuddAddNeg.c
        cuddBddCorr.c
-       cuddDecomp.c
+       cuddDecomp.cpp
```

Comment:
Why did you change the file extensions in CMakeLists.txt? It caused a cmake build crash on my mac.

However, the actual file names in CUDD folder remain unchanged?